README.md

```diff
@@ -88,12 +94,14 @@ Note the use of `isDryRunNetworkName` to prevent appending to the registry when
 
 ## 2. Merging network addresses into artifacts
 
-After Truffle compiles your smart contracts, you can merge the deployed addresses into the artifact by calling `apply-registry` with the path to the artifacts.
+After Truffle compiles your smart contracts, you can merge the deployed addresses into the artifacts by calling `apply-registry` from the terminal:
 
 ```sh
-$ apply-registry build/contracts
+$ apply-registry
 ```
 
+By default, apply-registry will use the truffle default directories: './networks' and './build/contracts'. If you need to customize this see [apply-registry options](#apply-registry-options).
```

Comment:
/networks isn't part of Truffle, so how about:

`By default, apply-registry will use the truffle artifact directory './build/contracts' and the network config directory './networks'.  If you need to customize this see [apply-registry options](#apply-registry-options).`